test/factories/CommandFactoryTest.ts

```diff
@@ -6,67 +6,67 @@ import MockCommand from "../MockCommand";
 import MockCommandWithAlias from "../MockCommandWithAlias";
 
 describe("CommandFactory", () => {
-    let factory: CommandFactory;
-    let commandName: string;
```

Comment:
The linter was complaining about some indentation problems (Expected 2 tabs, found 8 spaces). I didn't want to fix it because it would mess with the entire file and it was rendering normally in my editor. However, when I pushed the changes, the indenting on Github was awful (as you can see in the deleted lines). Hence, I autofixed all indentation problems. Let me know if this was a bad call.